runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessDetailsDialog/ProcessDetailsTabs/CredentialsTab.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from 'react';
+
+import { Box, Chip, Typography } from '@mui/material';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import { useDispatch, useSelector } from '#src-app/store';
+import { processActions, processSelector } from '#src-app/store/slices/Process';
+
+import {
+    DetailsInfoTabProps,
+    ProcessDetailsTab,
+} from '../ProcessDetailsDialog.types';
+
+export const CredentialsTab = ({ value, process }: DetailsInfoTabProps) => {
+    const { translate } = useTranslations();
+    const dispatch = useDispatch();
+    const {
+        draft: { credentials: processCredentials },
+    } = useSelector(processSelector);
+    const processId = process?.id;
+
+    useEffect(() => {
+        if (!processId) return;
```

Comment:
how this can happen? :D maybe I am missing something, but if we are able to click on details there has to be a process